types/react-devtools-inline/commons.d.ts

```diff
@@ -43,13 +43,13 @@ export type FetchFileWithCaching = (url: string) => Promise<string>;
 export type Context = FetchFileWithCaching | null;
 
 export interface DevtoolsProps {
-    bridge: FrontendBridge;
+    bridge?: FrontendBridge;
     browserTheme?: BrowserTheme | undefined;
     canViewElementSourceFunction?: CanViewElementSource | null | undefined;
     defaultTab?: TabID | undefined;
     enabledInspectedElementContextMenu?: boolean | undefined;
     showTabBar?: boolean | undefined;
-    store: Store;
+    store?: Store;
```

Comment:
not optional
![image](https://user-images.githubusercontent.com/56086371/196882764-1c86ab3d-6413-467b-bb9e-f0739af4eeb0.png)